dao/OrderDao.py

```
@@ -0,0 +1,267 @@
+            order_id: 订单ID
+            
+        Returns:
+            dict: 订单详细信息，包含订单基本信息和所有订单项
+        """
+        conn = None
+        cur = None
+        try:
+            conn = self.get_connection()
+            cur = self.get_cursor(conn)
+            
+            sql = sql.SQL("""
```

Ruff (0.11.9)

Local variable `sql` referenced before assignment

(F823)